app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.startswith("cp"):
```

Comment:
I think this condition is not working properly, what happens if the command is 'cpo'?)